Wox/MainWindow.xaml

```diff
@@ -56,7 +56,7 @@
     <Border Style="{DynamicResource WindowBorderStyle}" MouseDown="OnMouseDown" >
         <StackPanel Orientation="Vertical">
             <TextBox Style="{DynamicResource QueryBoxStyle}"
-                     Text="{Binding QueryText, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"
+                     Text="{Binding QueryText, Mode=TwoWay, Delay=100, UpdateSourceTrigger=PropertyChanged}"
```

Comment:
I am a bit conscious of this add here. I have occasionally run into the query not fully passed through/cutoff after I press enter in the mainwindow. Will do more testing.

Reply:
It should create a more streamlined experience since it will query less times, maybe we could lower it a little bit but 100 ms is considered a good value for these scenarios.

Reply:
I do get the intention behind this particular change. I have been testing just this change in the past couple of days and as suspected can actually confirm it has caused some missing query keystrokes (query search result cut off) in the execution if I type a long query very fast and hit enter. 

This is particular the case if I use Shell and WebSearch plugins and type long query commands fast, and in the previous Wox release I didn't have the issue. So after removing the delay has resolved the issue for me.

Design wise I think no delay for user input binding and as soon as a new query comes in then cancel the current running one.

Do you think we can just remove this particular change?

Reply:
sure 👍 if you saw it breaking current behavior I fully understand the reasoning